chromium/chromium-update.dockerfile

```diff
@@ -8,8 +8,10 @@ RUN sudo apt-get update -q && sudo apt-get upgrade -qy
 RUN cd /home/user/depot_tools \
  && git pull --rebase origin master
 
-# Update and rebuild Chromium's source code.
-RUN cd /home/user/chromium/src \
+# Remove the parallelism limited Ninja alias and
+# update and rebuild Chromium's source code.
+RUN unalias ninja \
```

Comment:
Nit: I'm not sure it's necessary to have two different behaviors (normal developers use half of cores vs update Dockerfile uses all cores), so I don't think you need to unalias here. Also maybe we should remove the `-j\`nproc\`` ninja parameter below.

Reply:
But then the image building will be slower, or is that not a concern?

Reply:
Actually, we import `*-update.dockerfile` into our admin interface, for when we want to update some project image by just building a few layers on top of it instead of pulling a brand new image (we stopped using `*-update.dockerfile` for most projects, but we still do it for Chrome, because we can't build the browser in the main Dockerfile as it makes CircleCI time out).

And since the `-j`nproc`` was OOM'ing OVH1, I manually edited it to `-j8` in our admin interface... So TL;DR we should only have one way to build Chrome, to be used everywhere by developers and CI alike.